styleguide/source/_patterns/01-atoms/welcome.md

```diff
@@ -1,4 +0,0 @@
----
-title: "Welcome"
```

Comment:
Needed to have these patterns around to make sure everything was functioning correctly. They're still needed presently as my work is still WIP.

Reply:
Ok